src/screens/DiscoveryScreenV2/elements/RecentSearchItem.js

```diff
@@ -25,7 +25,7 @@ const RecentSearchItems = (prop) => {
         borderless: false
       }}>
       <Text style={styles.text}>{text}</Text>
-      <IcArrowUpRight style={styles.icon} />
+      <IcArrowUpRight style={styles.icon} fill={COLORS.white2} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in this line is about adding a `fill` prop to the `IcArrowUpRight` component. This change seems fine as long as `COLORS.white2` is a valid color and the `IcArrowUpRight` component supports the `fill` prop.
```diff
- <IcArrowUpRight style={styles.icon} />
+ <IcArrowUpRight style={styles.icon} fill={COLORS.white2} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->